doc_swift_ui_gallery/swift_ui_gallery/swift_ui_gallery/primitives/text/typography/TextTypographyPage.swift

```diff
@@ -0,0 +1,349 @@
+        let endDate = Calendar.current.date(byAdding: .day, value: days, to: startDate)!
+        return DateInterval(start: startDate, end: endDate)
+    }
+    
+    var date: Date {
+        let components = DateComponents(year: 2023, month: 12, day: 7)
+        let calendar = Calendar.current
+        return calendar.date(from: components)!
+    }
+    
+    var body: some View {
+        ScrollView {
+            VStack(spacing: 24)  {
+                
+                
+                
+                // General styling
+                VStack(spacing: 24) {
+                    Text("Hello, world!")
+                    
+                    Text("Hello, world!")
+                        .bold()
+                    
+                    Text("Hello, world!")
+                        .italic()
+                    
+                    Text("Hello, world!")
+                        .background(.yellow)
+                    
+                    Text("Hello, world!")
+                        .strikethrough()
+                    
+                    Text("Hello, world!")
+                        .underline()
+                    
+                    Text("Hello, world!")
+                        .underline(pattern: .dot)
+                    
+                    Text("Hello, world!")
+                        .underline(pattern: .dash)
+                    
+                    Text("Hello, world!")
+                        .underline(pattern: .dashDot)
+                    
+                    Text("Hello, world!")
+                        .underline(pattern: .dashDotDot)
+                    
+                    Text("Hello, world!")
+                        .monospaced()
+                    
+                    Text("Hello, 12345!")
+                        .monospacedDigit()
+                    
+                    Text("Hello, world!")
+                        .kerning(-3)
+                    
+                    Text("Hello, world!")
+                        .kerning(3)
+                    
+                    Text("Hello, world!")
+                        .tracking(-3)
+                    
+                    Text("Hello, world!")
+                        .tracking(3)
+                    
+                    HStack(alignment: .top) {
+                        Text("Hello")
+                            .baselineOffset(-10)
+                            .border(Color.red)
+                        Text("Hello")
+                            .border(Color.green)
+                        Text("Hello")
+                            .baselineOffset(10)
+                            .border(Color.blue)
+                    }
+                    
+                    Text("Hello, world!")
+                        .textCase(.lowercase)
+                    
+                    Text("Hello, world!")
+                        .textCase(.uppercase)
+                    
+                    // TODO: text scale
```

Comment:
I don't know if this TODO was from me previously, or from you, but let's include a bit more context as to why the work hasn't been done yet, what it's blocked on, etc.